internal/kafka/internal/workers/kafka_mgrs/preparing_kafkas_mgr.go

```diff
@@ -103,6 +104,7 @@ func (k *PreparingKafkaManager) handleKafkaRequestCreationError(kafkaRequest *db
 			return errors.Wrapf(err, "Failed to update kafka %s in failed state", kafkaRequest.ID)
 		}
 		metrics.UpdateKafkaRequestsStatusSinceCreatedMetric(constants2.KafkaRequestStatusFailed, kafkaRequest.ID, kafkaRequest.ClusterID, time.Since(kafkaRequest.CreatedAt))
+		metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusFailed, kafkaRequest.ID, kafkaRequest.ClusterID)
```

Comment:
Same as above, let's only update this if the status is not already `Failed`. This is called on every reconcile so it will get updated every few seconds.